verification/curator-service/api/src/model/failed_attempts.ts

```diff
@@ -0,0 +1,84 @@
+import { Collection, ObjectId } from 'mongodb';
+import db from './database';
+
+const numberTimeLimiters = {
+    loginAttempt: {
+        maxNumberOfFailedLogins: 8,
```

Comment:
I think it would be best to make the request limit the same for both login and reset password